bindings/java/iota-wallet-java/lib/src/main/java/org/iota/types/events/transactionprogress/Broadcasting.java

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 IOTA Stiftung
```

Comment:
Can you put an underscore in `transactionprogress` in the folder name please ?

Reply:
no, thats not java standard :=)

Reply:
All of out other multi words folders have an underscore though :(

Reply:
That is a pretty stupid standard, I wouldn't mind not respecting it...

Reply:
agree, wouldn't mind not respecting it too

Reply:
Ive renamed it to just "transaction". to explain the names: WalletEvent is the wallet folder, TransactionProgressEvent goes in transaction.  Does that make sense? Theyre different enums after all.

Reply:
Yeah that's fine 👍🏻

Reply:
nice, thanks